app/views/search/index.html.erb

```diff
@@ -102,7 +102,10 @@
           <li class="result t-firm">
             <div class="result__heading">
               <%= heading_tag(firm.name, level: 2, class: 'result__heading-text t-name') %>
-              <p class="result__adviser-distance"><%= t('search.result.adviser') %> <%= number_to_human(firm.closest_adviser, significant: true) %> <%= t('search.result.miles_away') %></p>
+
+              <% if firm.closest_adviser %>
```

Comment:
Wondering if this conditional should be bound to the type of search rather than the presence of a value?